pkg/inject/sidecar_test.go

```diff
@@ -308,6 +309,7 @@ func TestSidecarOrderOfArguments(t *testing.T) {
 	containsOptionWithPrefix(t, dep.Spec.Template.Spec.Containers[1].Args, "--jaeger.tags")
 	containsOptionWithPrefix(t, dep.Spec.Template.Spec.Containers[1].Args, "--reporter.grpc.host-port")
 	containsOptionWithPrefix(t, dep.Spec.Template.Spec.Containers[1].Args, "--reporter.type")
+	containsAgentTag(t, dep.Spec.Template.Spec.Containers[1].Args, "container.name", "only_container")
```

Comment:
I think you can replace most of the new code in this test with this:

```golang
assert.Contains(t, util.FindItem("--jaeger.tags=", dep.Spec.Template.Spec.Containers[1].Args), "container.name=only_container")
```

Reply:
Ah, drat. I saw the first two comments, and then went ahead and refactored it, and ended up with this but slightly long-hand.